go.mod

```diff
@@ -40,35 +40,34 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.22.4
-	k8s.io/apiextensions-apiserver v0.22.4
-	k8s.io/apimachinery v0.22.4
-	k8s.io/cli-runtime v0.22.4
-	k8s.io/client-go v0.22.4
-	k8s.io/klog/v2 v2.30.0
-	k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65
-	sigs.k8s.io/controller-runtime v0.10.3
+	k8s.io/api v0.23.2
+	k8s.io/apiextensions-apiserver v0.23.2
+	k8s.io/apimachinery v0.23.2
+	k8s.io/cli-runtime v0.23.2
+	k8s.io/client-go v0.23.2
+	k8s.io/klog/v2 v2.40.1
+	k8s.io/kube-openapi v0.0.0-20220114203427-a0453230fd26
+	sigs.k8s.io/controller-runtime v0.11.0
 	sigs.k8s.io/kustomize/api v0.10.1
-	sigs.k8s.io/kustomize/kyaml v0.13.0
+	sigs.k8s.io/kustomize/kyaml v0.13.1
```

Comment:
This needs to be reverted to `v0.13.0` since it's not possible to use the current latest releases of `kustomize/api` and `kustomize/kyaml` in conjunction _(ref kubernetes-sigs/kustomize/issues/4409)_.

Reply:
cool, fixing it, thanks a lot for the review @zeborg !